IlmBase/CMakeLists.txt

```diff
@@ -150,11 +150,11 @@ FILE ( REMOVE ${ILMBASE_TMP_CONFIG} )
 UNSET ( ILMBASE_TMP_CONFIG )
 
 IF ( NOT WIN32 AND OPENEXR_BUILD_SHARED )
-  CREATE_LIBTOOL_FILE ( Half /lib )
-  CREATE_LIBTOOL_FILE ( Iex /lib )
-  CREATE_LIBTOOL_FILE ( IexMath /lib )
-  CREATE_LIBTOOL_FILE ( Imath /lib )
-  CREATE_LIBTOOL_FILE ( IlmThread /lib )
+  CREATE_LIBTOOL_FILE ( Half /${CMAKE_INSTALL_LIBDIR} )
+  CREATE_LIBTOOL_FILE ( Iex /${CMAKE_INSTALL_LIBDIR} )
+  CREATE_LIBTOOL_FILE ( IexMath /${CMAKE_INSTALL_LIBDIR} )
+  CREATE_LIBTOOL_FILE ( Imath /${CMAKE_INSTALL_LIBDIR} )
+  CREATE_LIBTOOL_FILE ( IlmThread /${CMAKE_INSTALL_LIBDIR} )
 ENDIF ()
```

Comment:
remove all of this. Creating libtool archives in CMake is a cardinal sin. Use pkgconfig and never, **ever** use Libtool archives. They're awful, clumsy, hard to use and CMake has no support for them.